tests/loggers/test_base.py

```diff
@@ -225,31 +225,6 @@ def validation_epoch_end(self, outputs):
     trainer.fit(model)
 
 
-def test_with_accumulate_grad_batches():
```

Comment:
Since the aggregation logic is in the training loop, I'm not sure this test makes sense anymore